presets.js

```diff
@@ -0,0 +1,169 @@
+export function getPresetDefinitions() {
+	const presets = {
+		preset_esc: {
+			type: 'button',
+			category: 'PPT',
+			name: 'Send ESC Command',
+			style: {
+				text: 'ESC',
+				size: '18',
+				color: '16777215',
+				bgcolor: '0',
+			},
+			steps: [
+				{
+					down: [
+						{
+							actionId: 'ESC',
+						},
+					],
+					up: [],
+				},
+			],
+		},
+		preset_next: {
+			type: 'button',
+			category: 'PPT',
+			name: 'Send NEXT Command',
+			style: {
+				text: '>\nNext',
+				size: '14',
+				color: '16777215',
+				bgcolor: '25600',
+			},
+			steps: [
+				{
+					down: [
+						{
+							actionId: 'NEXT',
+						},
+					],
+					up: [],
+				},
+			],
+		},
+		preset_previous: {
+			type: 'button',
+			category: 'PPT',
+			name: 'Send PREVIOUS Command',
+			style: {
+				text: '<\nPrevious',
+				size: '14',
+				color: '16777215',
+				bgcolor: '9109504',
+			},
+			steps: [
+				{
+					down: [
+						{
+							actionId: 'PREVIOUS',
+						},
+					],
+					up: [],
+				},
+			],
+		},
+		preset_fechar_ppt: {
+			type: 'button',
+			category: 'PPT',
+			name: 'Send FECHAR_PPT Command',
+			style: {
+				text: 'PPT\nEXIT',
+				size: '18',
+				color: '16777215',
+				bgcolor: '0',
+			},
+			steps: [
+				{
+					down: [
+						{
+							actionId: 'FECHAR_PPT',
+						},
+					],
+					up: [],
+				},
+			],
+		},
+		preset_fechar_video: {
+			type: 'button',
+			category: 'VIDEO',
+			name: 'Send FECHAR_VIDEO Command',
+			style: {
+				text: 'VIDEO EXIT',
+				size: '18',
+				color: '16777215',
+				bgcolor: '0',
+			},
+			steps: [
+				{
+					down: [
+						{
+							actionId: 'FECHAR_VIDEO',
+						},
+					],
+					up: [],
+				},
+			],
+		},
+	}
+
+	// Generate PPT command presets
```

Comment:
See my comment at the 'send' action.

Reply:
I don't know how to solve this part. I would like help

Reply:
I already wrote a lot of comments with a lot of help. You can find the source code of over 600 Companion modules here at GitHub. Look at them, learn how the others are doing it. Not just copy one and think this will fit you completely. There is plenty of documentation in the Wiki. Read it. There is tons of information about javascript on the internet. Learn it.
If you still don't understand what to do then, I think it would be best if you hire someone to do it for you.